### ISSUE TITLE
Suggest mentioning @Value.Enclosing in "Generated source file name collission" error

### ISSUE THREAD

**Issue by nezda** [immutables/immutables#334]
I recently hit the error below when I had 2 `static` inner `@Value.Immutable` classes in different enclosing classes in the same package and it took me a while to find the solution to my problem: [`@Value.Enclosing`](http://immutables.github.io/style.html#nesting) (awesome feature!).  I suggest mentioning that feature in the error message, e.g., "@Value.Enclosing might help you fix this" or something)

```
[ERROR] Generated source file name collission. Attempt to overwrite already generated file: com.foo.ImmutableBar, javax.annotation.processing.FilerException: Attempt to recreate a file for type com.foo.ImmutableBar
```

(also collission -> colission)

**Comment by elucash** [comment]
Will add this. Good idea! Thank you

**Comment by elucash** [comment]
Released in 2.2.1 Thank you!
